### DISCUSSION TITLE
What if you were able to set a default player (Scratch, TurboWarp, etc)?

### DISCUSSION THREAD

**Discussion by ghost** [ScratchAddons/ScratchAddons#1430] [Ideas]
I mean like that would be pretty cool I guess

the settings would probably look like this
![lol look at it](https://user-images.githubusercontent.com/77708165/105999564-78418f00-6073-11eb-9162-cb1ed8432335.png)

**Comment by WorldLanguages** [comment]
This almost became an addon for v1.0 but it was badly done so I just decided to drop it - and I don't think anyone worked on it ever again.
